client-projects/the-lumery/scss/hero-heading-animation.scss

```diff
@@ -15,6 +15,13 @@
       font-size: 4.75rem;
     }
   }
+  .hero--secondary & {
+    font-size: 6.25rem;
+
+    @media only screen and (max-width: 479px) {
```

Comment:
Maybe these breakpoint values could be sass vars? (noting that CSS vars can't be used here)